public_html/launch.php

```diff
@@ -1,379 +1,469 @@
+      $form_param_name .
+      '_false" name="' .
+      $form_param_name .
+      '" ' .
+      $required .
+      ' value="false">
+                <label class="form-check-label" for="' .
+      $form_param_name .
+      '_false">False</label>
             </div>
         </div>';
-    }
-
-    // enum input
-    if (array_key_exists('enum', $param) && count($param['enum']) > 0) {
-        $input_el = '<select class="form-select" id="' . $form_param_name . '" name="' . $form_param_name . '" ' . $required . '>';
-        $input_el .= '<option value="">[ Select an option ]</option>';
-        foreach ($param['enum'] as $option) {
-            $input_el .= '<option ' . ($value == $option ? 'selected' : '') . ' value="' . $option . '">' . $option . '</option>';
-        }
-        $input_el .= '</select>';
-    }
-
-    // Build HTML
-    return '
-    <div class="form-group param-form-group ' . $hide_class . '" id="' . $param_id . '_group">
+  }
+
+  // enum input
+  if (array_key_exists('enum', $param) && count($param['enum']) > 0) {
+    $input_el =
+      '<select class="form-select" id="' . $form_param_name . '" name="' . $form_param_name . '" ' . $required . '>';
+    $input_el .= '<option value="">[ Select an option ]</option>';
+    foreach ($param['enum'] as $option) {
+      $input_el .=
+        '<option ' . ($value == $option ? 'selected' : '') . ' value="' . $option . '">' . $option . '</option>';
+    }
+    $input_el .= '</select>';
+  }
+
+  // Build HTML
+  return '
+    <div class="form-group param-form-group ' .
+    $hide_class .
+    '" id="' .
+    $param_id .
+    '_group">
         <div class="input-group">
-            <label class="input-group-text font-monospace" for="' . $form_param_name . '">' . $fa_icon . $dash_param_id . $required_asterisk . '</label>
-            ' . $input_el . $help_text_btn . $validation_text . '
+            <label class="input-group-text font-monospace" for="' .
+    $form_param_name .
+    '">' .
+    $fa_icon .
+    $dash_param_id .
+    $required_asterisk .
+    '</label>
+            ' .
+    $input_el .
+    $help_text_btn .
+    $validation_text .
+    '
         </div>
-        ' . $description . $help_text . '
+        ' .
+    $description .
+    $help_text .
+    '
     </div>';
 }
 
 // Got this far without printing JSON - build web GUI
 $title = 'Launch pipeline';
 $subtitle = 'Configure workflow parameters for a pipeline run.';
 $import_schema_launcher = true;
-include('../includes/header.php');
+include '../includes/header.php';
 
 if (count($error_msgs) > 0) {
-    echo '<div class="alert alert-danger">' . implode('<br>', $error_msgs) . '</div>';
+  echo '<div class="alert alert-danger">' . implode('<br>', $error_msgs) . '</div>';
 }
 
-if (!$cache) {
-
-    /////////////////////////////////////////////////////////////
-    // EMPTY - NO CACHE ID
-    /////////////////////////////////////////////////////////////
-?>
+if (!$cache) {<?php
```

Comment:
Gonna need help from someone who knows what they're doing here.

Reply:
that's indeed a weird bug(?)...

Reply:
How about we put the comment block above the `if` statement and then change this to just:

```suggestion
if (!$cache) { ?>
```

Not ideal that this linting bug is there, but simple refactoring may get around it..

Reply:
Sounds good to me. I think it's just an edge case that needs to be fixed manually.